backend/app/api/routes/content.py

```
@@ -547,4 +553,168 @@ async def stream_analysis() -> AsyncGenerator[str, None]:
+@router.post(
+    "/{id}/share",
+    response_model=ContentSharePublic,
+    status_code=status.HTTP_201_CREATED,
+    summary="Create a Share Link for a Content Item",
+    description="Generates a shareable link for the specified content item. Requires ownership.",
+)
+def create_share_link_endpoint(
+    *,
+    session: SessionDep,
+    current_user: CurrentUser,
+    id: uuid.UUID = Path(..., description="ID of the content item to share"),
```

Pylint (3.3.7)

[warning] Redefining built-in 'id'

(W0622)